DigitalLearningSolutions.Data/Models/Tracker/GetObjectiveArrayData.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace DigitalLearningSolutions.Data.Models.Tracker
+{
+    public class GetObjectiveArrayData : ITrackerEndpointDataModel

```

Comment:
`GetObjectiveArrayData` is slightly confusing because it can be interpreted as a verb.
What about `TrackerObjectiveArray`? `ObjectiveArrayForTracker`?